apps/kita/app/Http/Controllers/Auth/LoginController.php

```diff
@@ -20,6 +23,30 @@ class LoginController extends Controller
     */
 
     use AuthenticatesUsers;
+    public function logout(Request $request)
+    {
+        \Auth::guard('members')->logout();
```

Comment:
```suggestion
        $this->guard()->logout();
```

こっちのほうが重複なくていいかなって思います。

Reply:
AdminのLoginに書いたけどシンプルにできるかと思います。